haas/api.py

```diff
@@ -510,6 +510,19 @@ def headnode_detach_network(headnode, hnic):
                             # Network Code #
                             ################
 
+@rest_call('GET', '/networks')
+def list_networks():
+    """List all networks.
+    Returns a JSON array of strings representing a list of projects.
+    Example:  {"channels": ["101", "107"], "name": ["net-05", "qnet-00"]}
+    """
+    db = model.Session()
+    networks = db.query(model.Network).all()
+    result = {
+        'channels': [n.network_id for n in networks],
+        'names': [n.label for n in networks],
```

Comment:
Thanks for the useful note. I am bit confused on how to show project within list_network. We currently don't have a n.project to query projects that connected to this network. I think we do have access_id which accentually just project_id, right? 

For changing "access" to "project", it needs me to change the unit test as well, and it causes lots of failure. I will try to fix them.